docs/command-reference/stream/xsetid.md

```diff
@@ -10,17 +9,11 @@ import PageTitle from '@site/src/components/PageTitle';
 
 ## Syntax
 
-	XSETID key last-id
+    XSETID key last-id [ENTRIESADDED entries-added] [MAXDELETEDID max-deleted-id]
 
-**Time Complexity:** O(1)
+**Time complexity:** O(1)
 
 **ACL categories:** @write, @stream, @fast
 
-**XSETID** sets the last id of the specified stream. **<key\>**
-must exists before executing the command. The **<last-id\>** can't
-be smaller than target stream top entry.
-
-## Return
-
-[Simple string reply](https://redis.io/docs/reference/protocol-spec/#simple-strings):
-**OK** on success.
+The `XSETID` command is an internal command.
+It is used to replicate the last delivered ID of streams.
```

Comment:
we had `xesid.md` in both /stream and /pubsub :face_exhaling: 

@Niennienzz let me know which one you would like me keep

Reply:
For example this one is missing the ##return below

Reply:
Yeah I see, we should keep those ones in the `/stream`. Thanks!